ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```diff
@@ -251,6 +267,12 @@ spec:
         # Adding a commit hash as a suffix to the image tag
         - name: index-image-destination-tag-suffix
           value: "-$(params.git_commit)"
+        - name: upgrade-graph-mode
+          value: "$(tasks.read-config.results.upgrade-graph-mode)"
+        # Production IIB instance doesn't support the upgrade path feature yet.
+        # TODO: switch it to PROD when the feature is available
+        - name: iib_url
```

Comment:
Should we create a Task ticket for this?

Reply:
I am in sync with a iib developer who takes care of that. I'll change it once it is ready.